test/correctness/gpu_data_flows.cpp

```diff
@@ -6,10 +6,6 @@ using namespace Halide;
 int main(int argc, char **argv) {
 
     Target target = get_jit_target_from_environment();
-    if (!target.has_gpu_feature() && !target.has_feature(Target::OpenGLCompute)) {
```

Comment:
Yeah, it's a bit weird. I thought about it, but it seems harmless to just let them run anyways, even though they aren't exercising much unless there is a device (GPU, Hexagon) involved.